editor/assets/effects/pipeline/deferred-lighting.effect

```diff
@@ -94,18 +94,19 @@ CCProgram lighting-fs %{
     #endif
 
     s.albedo = albedoMap;
-    s.position = positionMap.xyz;
+    vec3 position = positionMap.xyz;
+    s.position = position;
```

Comment:
HIGHP_VALUE_TO_STRUCT_DEFINED(position, s.position);

Reply:
careful still you careful
![image](https://user-images.githubusercontent.com/15099632/147308544-15da7796-bc98-4aab-a328-a81be63001a5.png)